### DISCUSSION TITLE
homebrewのインストール

### DISCUSSION THREAD

**Discussion by Tomohiro-gh** [nig-sc/nigsc_homepage2#312] [Q&A] [answered]
いつも大変お世話になっております．

スパコンにhomeberwを入れることはできるのでしょうか？

homebrewのサイトに従いhomebrewを入れようとすると管理者権限の問題でエラーが出てしまいます．
https://brew.sh/
```
$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

==> Checking for `sudo` access (which may request your password)...
Sorry, user xxxxx may not run sudo on at137.
Insufficient permissions to install Homebrew to \"/home/linuxbrew/.linuxbrew\" (the default prefix).

Alternative (unsupported) installation methods are available at:
https://docs.brew.sh/Installation#alternative-installs

Please note this will require most formula to build from source, a buggy, slow and energy-inefficient experience.
We will close any issues without response for these unsupported configurations.
```

もし方法があればご教示いただければ幸いです．
どうぞよろしくお願い致します．

**Comment by sc-fs** [comment, marked as answer]
homebrew (旧linuxbrew)のスパコン環境での利用を過去動作検証しましたが、
共用環境へインストールが可能なアカウントが、管理者または環境を構築したアカウントに限定
されてしまうため、誰もがインストールできるものではありません。
そのため、共用環境で使用するのに適していないと判断しており、スパコンへのインストールもできません。
遺伝研スパコンで利用可能なパッケージマネージャをスパコンHPでも紹介しておりますので、こちらの利用をご検討いただけばと存じます。
https://sc.ddbj.nig.ac.jp/software/#%E3%83%91%E3%83%83%E3%82%B1%E3%83%BC%E3%82%B8%E3%83%9E%E3%83%8D%E3%83%BC%E3%82%B8%E3%83%A3
